Automattic/wp-calypso
# composite-checkout: Remove usePrimaryDispatch/usePrimarySelect from public api

Because components outside of this package cannot add selectors or
actions to the primary store, there's no reason to expose these hooks.

#### Testing instructions

Verify that neither `usePrimaryDispatch` nor `usePrimarySelect` are used anywhere outside of the `composite-checkout` package.
